go.mod

```diff
@@ -33,10 +33,10 @@ require (
 	github.com/filecoin-project/go-cbor-util v0.0.1
 	github.com/filecoin-project/go-commp-utils v0.1.3
 	github.com/filecoin-project/go-crypto v0.0.1
-	github.com/filecoin-project/go-data-transfer v1.12.1
+	github.com/filecoin-project/go-data-transfer v1.13.0
 	github.com/filecoin-project/go-fil-commcid v0.1.0
 	github.com/filecoin-project/go-fil-commp-hashhash v0.1.0
-	github.com/filecoin-project/go-fil-markets v1.14.1
+	github.com/filecoin-project/go-fil-markets v1.14.2-0.20220115005823-180ae44bbc36
```

Comment:
Should be a tag

Reply:
@hannahhoward context here is just that we're trying to diligently avoid commit deps in lotus because it's breaking a little too often and pissing people off